akka-actor/build.sbt

```diff
@@ -1,10 +1,10 @@
-import akka.{ AkkaBuild, Formatting, OSGi, Dependencies, Version }
+import akka.{ AkkaBuild, Formatting, OSGi, Dependencies, VersionGenerator }
 
 AkkaBuild.defaultSettings
 Formatting.formatSettings
 OSGi.actor
 Dependencies.actor
```

Comment:
That’s weird - conflicted with what, do you know? Just the class that we have callled like that in class path...?

Reply:
Right: to the generated `akka.Version` that's pulled in as a dependency of sbt-web